app/models/product.rb

```diff
@@ -0,0 +1,6 @@
+class Product < ApplicationRecord
+
```

Comment:
Nitpick: 
a) I can see 4 space indentation, lets keep it to two.
b) Extra line.